Sources/AblyChat/ChatAPI.swift

```diff
@@ -66,12 +66,12 @@ internal final class ChatAPI: Sendable {
 
     // (CHA-M3) Messages are sent to Ably via the Chat REST API, using the send method.
     // (CHA-M3a) When a message is sent successfully, the caller shall receive a struct representing the Message in response (as if it were received via Realtime event).
-    internal func sendMessage(roomID: String, params: SendMessageParams) async throws(InternalError) -> Message {
+    internal func sendMessage(roomName: String, params: SendMessageParams) async throws(InternalError) -> Message {
```

Comment:
That's not a good practice, `sendMessage: ...` implies that the first parameter shoud be `Message`, So I would suggest this signature:
```
func sendMessage(_ message: Message, roomName: String, params: SendMessageParams)
```
ditto other methods.

Reply:
I left with `roomName` as a first argument, because it's internal API, that using in one place, and `sendMessage` has `roomName` as the first argument. I don't think `func updateMessage(_ message: Message, roomName: String, params: SendMessageParams)` much more readable, I am happy if we rename it later, just don't want to overcomplicate this PR. It think we will delete message in the `updateMessage` anyway :)